packages/ui/src/forum/modals/PostActionModal/PostActionSignModal.tsx

```diff
@@ -40,33 +63,50 @@ export const PostActionSignModal = ({ transaction, service, controllerAccount, a
   const signDisabled = !state.matches('prepare') || !hasFunds
 
   return (
-    <TransactionModal onClose={hideModal} service={service}>
-      <ModalBody>
-        <RowGapBlock gap={24}>
-          <RowGapBlock gap={16}>
-            <TextMedium>{actionText}</TextMedium>
-            <TextMedium>
-              A fee of <TokenValue value={paymentInfo?.partialFee} /> will be applied to the transaction.
-            </TextMedium>
+    <>
+      <TransactionModal onClose={hideModal} service={service}>
+        <ModalBody>
+          <RowGapBlock gap={24}>
+            <RowGapBlock gap={16}>
+              <TextMedium>You intend to {action} your post.</TextMedium>
```

Comment:
```suggestion
              <TextMedium>You intend to {action} your post.</TextMedium>
```

We shouldn't do that for text fields. ATM we're using English, but there might be a need to translate the app.

Because of this I'd use two separate strings or just allow to pass a text as before. The reason for this is that it will be easier to translate a whole text.